### PR TITLE
Revised PWA docs per June 22 meeting

### PR DESCRIPTION
**Pull request by mikehoffms**
Changes discussed in the meeting and made here:
* Move Push Notifications section of get-start to become separate article.
* Expand and rearrange TOC.
* etc.
* Pending: copy code from workshop to top of article, above Shortcuts section.

Motivation for creating this Draft PR: Patrick is heavily revising PWA docs in his Draft PR and we need to monitor both PRs.

### CODE REVIEW

**Review by captainbrosset — COMMENTED** [review]
Those content changes look good to me. We'll need to think about how we can bring them into PR #1429 later, since I'm moving/changing a lot of the content in that PR.

File: microsoft-edge/progressive-web-apps-chromium/pwabuilder.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+description: PWABuilder.
+title: PWABuilder
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 09/16/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+keywords: progressive web apps, PWA, Edge, Windows, PWABuilder, web manifest, service worker, push
+---
+# PWABuilder
```
Comment (captainbrosset) [inline microsoft-edge/progressive-web-apps-chromium/pwabuilder.md:11]: Can you tell me more about the rationale behind adding a new page for PWABuilder here? 
We already have a section about using PWABuilder in other places, and this short page doesn't seem to be adding a lot.

**Comment by captainbrosset** [comment]
> We'll need to think about how we can bring them into PR #1429 later, since I'm moving/changing a lot of the content in that PR.

I actually did this. My last commit in the feature branch now contains most of your changes applied to my branch: https://github.com/MicrosoftDocs/edge-developer/pull/1429/commits/d2e6d0d746c36017dd42fad3a797f208afb1b75e

**Comment by mikehoffms** [comment]
These changes were merged into PR 1429 https://github.com/MicrosoftDocs/edge-developer/pull/1429

**Pull request closed without merging**